apps/financial/src/lib/actions.ts

```
@@ -91,7 +91,11 @@ export async function deleteInvoice(id: string) {
 
 export async function authenticate(prevState: string | undefined, formData: FormData) {
     // TODO implement authentication
-    redirect('/dashboard');
+    return redirect('/dashboard');
+
+
+console.log();
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)